rosidl_typesupport_opendds_cpp/resource/srv__type_support.cpp.em

```diff
@@ -119,45 +119,49 @@ void * create_requester__@(service.namespaced_type.name)(
     @(__dds_response_wrapper_msg_type)
   >;
 
+  if (request_topic_str == nullptr || response_topic_str == nullptr || dds_participant==nullptr
+      || dds_publisher == nullptr || dds_subscriber == nullptr) {
+     RMW_SET_ERROR_MSG("Invalid parameters for create_requester");
+     return nullptr;
+  }
+
   @# Register TypeSupports
   @(__dds_msg_typesupport_type)RequestWrapperTypeSupport_var tsRequest =
     new @(__dds_msg_typesupport_type)RequestWrapperTypeSupportImpl;
 
   if (tsRequest->register_type(dds_participant, "@(__dds_request_wrapper_msg_type)_") != DDS::RETCODE_OK) {
-    RMW_SET_ERROR_MSG("request register_type for requester failed");
+    RMW_SET_ERROR_MSG("Request register_type for requester failed with @(__dds_request_wrapper_msg_type)_ type");
     return nullptr;
   }
 
   @(__dds_msg_typesupport_type)ResponseWrapperTypeSupport_var tsResponse =
     new @(__dds_msg_typesupport_type)ResponseWrapperTypeSupportImpl;
 
   if (tsResponse->register_type(dds_participant, "@(__dds_response_wrapper_msg_type)_") != DDS::RETCODE_OK) {
-    RMW_SET_ERROR_MSG("response register_type for requester failed");
+    RMW_SET_ERROR_MSG("Response register_type for requester failed with @(__dds_response_wrapper_msg_type)_ type");
     return nullptr;
   }
 
   @# Create Topics
-  CORBA::String_var type_name = tsRequest->get_type_name();
   DDS::Topic_var request_topic = dds_participant->create_topic(request_topic_str,
-                                                      type_name,
+                                                      "@(__dds_request_wrapper_msg_type)_",
```

Comment:
This change is not what's described in the title of the PR

Reply:
Changed the title to reflect this